### PR TITLE
Add notes about build notifier

### PR DESCRIPTION
**Pull request by do3cc**
Put Jenkins higher above, because it is not an afterthought.
Reword some phrases to not server as a cheap excuse not to write tests.
@tisto ?

### CODE REVIEW

File: docs/intro.rst [hunk 1]
```diff
@@ -202,62 +236,6 @@ actually merge it to the 4.2 release branch. Let's do that now::
   > git merge my-awesome-feature-4.2
 
 
-Committing to Products.CMFPlone
```
Comment (tisto) [inline docs/intro.rst:205]: @do3cc Why did you delete the entire section?

Comment (do3cc) [inline docs/intro.rst:205]: It was in there twice
